data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-creation-v5.1.xml

```diff
@@ -3,18 +3,106 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
 
+    <property name="json.column.type" value="JSONB" dbms="postgresql"/>
+
+    <!-- Please note that HSQLDB schema is supported for integration test purposes only. -->
+    <property name="json.column.type" value="CLOB" dbms="hsqldb"/>
+
     <changeSet author="R3.Corda" id="ledger-utxo-creation-v5.1">
```

Comment:
According to the [Liquibase Best Practice](https://www.liquibase.org/get-started/best-practices) guide, we should generally do a single change per changeset. I believe that is what we generally do elsewhere (and certainly what we did for the notary in C4).

Reply:
I just copied other changesets to be honest. 😅 `vnode-vault-creation` has tons of changes in one big changeset. I thought that's the practice we are following.

Reply:
I remember that one change per changeset used to be the practice for C4 but it seems like we moved away from that (not sure if intentionally)?

Reply:
The rationale is that each changeset represents an atomic step that can be rolled back independently if possible, the more complexity / steps in a changeset, the more scope there is for things to go wrong.

If you remember this was particularly important when we were working with CockroachDB because that cannot run changsets in transactions IIRC - so it was super important to keep them simple to minimise the chance of failure (which would then potentially need a manual fix)

Reply:
Are we changing this part based on @relyafi's comment or not? It doesn't look like it has changed at least, I don't have strong opinions myself, so happy to go either way.

Reply:
I think the idea is that we merge this as-is then rework how changesets are defined in the whole codebase (other areas of the code as well)